contracts/test/helpers/TestBaseWorkflow.t.sol

```diff
@@ -187,6 +244,7 @@ abstract contract TestBaseWorkflow is DSTest {
         _multisig
       )
     );
+    _terminals.push(_jbERC20PaymentTerminal);
```

Comment:
I don't think the eth terminal is needed here (this test base was for jbx-v2, and is probably having a lot of unneeded parts in this context)